modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -215,10 +229,13 @@ class {{classname}}(object):
         if not isinstance(other, {{classname}}):
             return False
 
-        return self.__dict__ == other.__dict__
+        return self.to_dict() == other.to_dict()
```

Comment:
What's the reason to use `to_dict()` instead of `__dict__`?

Reply:
So far configuration was not part of the model, now configuration is. So if the user does not provide configuration. The configuration is initialized since two objects have different configuration self.__dict__ == other.__dict__ will return false even though its the same object.

Test("abc") will not be equal to Test("abc") since configuration objects inside both are different. This change is required to prevent this behaviour

Reply:
@slash-arun thanks for the explanation. I wonder if you can add test cases to https://github.com/OpenAPITools/openapi-generator/tree/master/samples/client/petstore/python/tests to cover the issue moving forward.

Reply:
Sure, will work on the test cases